partner-management-default.properties

```diff
@@ -67,7 +67,7 @@ hibernate.generate_statistics=false
 spring.datasource.initialization-mode=never
 
 ## This property is used by kernel-authcodeflowproxy-api to check request is coming from allowed urls not.
-auth.allowed.urls=https://${mosip.pmp.host}/
+auth.allowed.urls=https://${mosip.pmp.host}/,https://${mosip.pmp.reactjs.ui.host}/
```

Comment:
I believe this change is related to the latest revamp, is that in the scope for the beta.1 release ?

Reply:
Lets understand this for all the changes this file

Reply:
Requested @mayuradesh to review this.

Reply:
This is only needed for revamp code

Reply:
Removed.